problems/2.functions.js

```diff
@@ -1,41 +1,57 @@
 /** Logs a custom greeting to the console in the form "Hello Avery!"
-* @param {string} name - The name of the user to greet
-* @returns {null}
-*/
+ * @param {string} name - The name of the user to greet
+ * @returns {null}
+ */
 function logGreeting() {
+    let logGreeting = "Hello Ben!"
```

Comment:
This will always log "hello ben!" We need to pass in the aragument name so we can greet anybody.

Reply:
copy! Thank you